great_expectations/data_context/store/query_store.py

```diff
@@ -23,7 +23,7 @@ class SqlAlchemyQueryStore(Store):
     """SqlAlchemyQueryStore stores queries by name, and makes it possible to retrieve the resulting value by query
     name."""
 
-    _key_class = StringKey
+    _key_class = StringKey  # type: ignore[assignment] # StringKey is a DataContextKey??
```

Comment:
`StringKey` inherits from `DataContextKey` (which I believe is the base class for all store keys)